File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        k=0
        m=0
        for i in range(len(nums)):
            if nums[i]==1:
                k+=1
                m=max(m,k)
            elif nums[i]!=1:
                k=0
        return m

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let maxi=0;
    let count=0;
    for (let num of nums){
        if (num===1){
            count++;
            maxi=Math.max(maxi,count)
        }else{
            count=0;
        }
    }
    return maxi;
};